src/react-access-context.js

```diff
@@ -0,0 +1,43 @@
+import {Component, PropTypes} from 'react';
+import _ from 'lodash';
```

Comment:
might even be worthwhile to write a quick version too, for the zero dep badge in the readme